remotespark/livyclientlib/livysession.py

```diff
@@ -12,7 +13,8 @@
 from .livyreliablehttpclient import LivyReliableHttpClient
 
 
-class LivySession(object):
+
```

Comment:
Please revert all whitespace changes on this file. It's no longer PEP compliant.